docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -45,6 +45,7 @@ In the logical diagram above, each type of consumer is shown as a single box. In
 ### Additional considerations
 
 - The amount of data to include in an event can be a significant consideration that affects both performance and cost. Putting all the relevant information needed for processing in the event itself can simplify the processing code and save additional lookups. Putting the minimal amount of information in an event, like just a couple of identifiers, will reduce transport time and cost, but requires the processing code to look up any additional information it needs. For more information on this, take a look at [this blog post](https://particular.net/blog/putting-your-events-on-a-diet).
+- While a request is only visible to the receiving component, events are usually visible to all components in the system, even if they do not or are not meant to consume them. Be careful what information you include in events, since they are published to the entire system.
```

Comment:
```suggestion
- While a request is only visible to the request-handling component, events are often visible to multiple components in a workload, even if those components don't or aren't meant to consume them. Operating with an "assume breach" mindset, be mindful of what information you include in events to prevent unintended information exposure.
```